deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml

```diff
@@ -27,9 +27,13 @@ restAPIConfig:
   opsManagerUserName: Turbo_username
   opsManagerPassword: Turbo_password
 
-# Enabling targetConfig and providing a value will give a name to your cluster
-#targetConfig:
-#  targetName: Name_Each_Cluster
+# Specify a targetName only will register a probe with type Kubernetes-<targetName>, as well as adding your
+# cluster as a target with the name Kubernetes-<targetName>.
+# If you want to register a probe without adding your cluster as a target, specify a targetType only. The probe
+# will appear as a Cloud Native probe in the UI with a type Kubernetes-<targetType>.
+targetConfig:
```

Comment:
so the problem with this, that instead of a generated clustername - every kubeturbo instance/every cluster would be named the same - confusing - I think we should keep the default behaviour and let the customer name their clusters